crates/api_models/src/payment_methods.rs

```diff
@@ -459,8 +457,6 @@ pub struct RequestPaymentMethodTypes {
 #[derive(Debug, Clone, serde::Serialize, Default, ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct PaymentMethodListRequest {
-    #[serde(skip_deserializing)]
-    pub customer_id: Option<CustomerId>,
```

Comment:
Check with the @juspay/hyperswitch-sdk-devs folks once to ensure that this field is not being passed. If it is removed here, then any client who was passing this field will fail to work because of `#[serde(deny_unknown_fields)]`